component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateUtil.java

```diff
@@ -251,6 +251,31 @@ public static UserRealm getUserRealm(String username) throws AuthenticationFaile
         return userRealm;
     }
 
+    /**
+     * Get user realm by tenant domain.
+     *
+     * @param tenantDomain tenant domain
+     * @return user realm
+     * @throws AuthenticationFailedException if error occurred while getting user realm.
+     */
+    public static UserRealm getUserRealmByTenantDomain(String tenantDomain) throws AuthenticationFailedException {
+
+        UserRealm userRealm = null;
+        log.debug(String.format("Getting userRealm for tenantDomain: %s", tenantDomain));
```

Comment:
No need to user String.format since only one param is passed

Reply:
Resolved with 5f0e1d972a4fb2c2ddf7699da6e7935af4b44c31